lib/xandra/authenticator.ex

```diff
@@ -0,0 +1,3 @@
+defmodule Xandra.Authenticator do
+  @callback response_body(Keyword.t) :: String.t
```

Comment:
We tend to use `binary` when something is not a readable string, but here we should use iodata (http://elixir-lang.org/getting-started/io-and-the-file-system.html#iodata-and-chardata) like we do in the rest of the encoding API.